main.go

```diff
@@ -168,35 +185,20 @@ func summaryMessage(base, head int) string {
 	return fmt.Sprintf("Coverage increased by %6.1f%%. Keep it up :medal_sports:", float64(head-base)/100)
 }
 
-func relativePackage(pkg, root string) string {
-	if strings.HasPrefix(pkg, root) {
-		return "./" + strings.TrimPrefix(pkg, root)
-	}
-	return pkg
-}
-
 func getModulePackageName() string {
 	f, err := os.ReadFile("go.mod")
 	if err != nil {
+		// unable to determine package name
 		return ""
 	}
+
 	// found it, stop searching
-	return string(modRegex.FindSubmatch(f)[1]) + "/"
+	modRegex := regexp.MustCompile(`module +([^\s]+)`)
+	return string(modRegex.FindSubmatch(f)[1])
 }
 
-var modRegex = regexp.MustCompile(`module ([^\s]*)`)
```

Comment:
Might be worth keeping it here because it's safe for concurrent use and it's only compiled once per application run.

On the flipside, I'm not sure we actually invoke `getModulePackageName` more than once per run, so potentially doesn't matter in this case.

Reply:
Yeah it's only used one - so in this case I decided to move it inside the function. 👍